static/_redirects

```diff
@@ -441,6 +441,8 @@
 ## Dynamic redirects (with wildcards and splats)
 # Note: Dynamic redirects must come after static redirects
 /data-vis/* /guides/tables/ 301
+/ref/python/classes/* /ref/python/experiments/:splat 301
```

Comment:
Ah, hmmm...this might be messier than I think it was...
```
/ref/python/sdk/classes/artifacts -> /ref/python/classes/artifacts
/ref/python/sdk/classes/settings -> /ref/python/classes/settings
/ref/python/sdk/classes/run ->  /ref/python/classes/run
```

Reply:
Actually, the current locations are indeed /ref/python/experiments/ 
https://github.com/wandb/docs/tree/main/content/en/ref/python/experiments

I think Line 445 is correct as proposed

Reply:
Sorry sorry. This was a typo 🤦 Updated comment.